WalletWasabi.Fluent/ZXing/.editorconfig

```diff
@@ -0,0 +1,31 @@
+root = true
+
+# For all files.
+[*]
+
```

Comment:
An alternative way may be (untested) to just put here:

`generated_code = true`

instead of all the rules below

Reply:
Thanks